modules/migration-rsync-manually-overriding.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * migration_toolkit_for_containers/installing-mtc.adoc
+// * migration_toolkit_for_containers/installing-mtc-restricted.adoc
+
+:_content-type: PROCEDURE
+[id="migration-rsync-manually-overriding_{context}"]
+== Manually overriding default non-root operation for data transfer
+
+Although running Rsync pods as non-root user works in most cases, data transfer might fail when when you run workloads as root user on the source side. {mtc-short} provides two ways to manually override default non-root operation for data transfer: configuring all migrations to run an Rsync pod as root on the destination cluster for all migrations or to run an Rsync pod as root on the destination cluster per migration.
+
+In both cases, you must set the following labels on the source side of any namespaces that are running workloads with higher privileges prior to migration: `enforce`, `audit`, and `warn.`
+
+.Procedure
+
+* Prior to migration, if you are running workloads as root on the source side of a migration, set the `enforce`, `audit`, and `warn` labels on the source side of the Rsync pod as follows:
```

Comment:
"set the `enforce`, `audit`, and `warn` labels on the source side of the Rsync pod as follows"

This line doesn't make it very clear about where to set those labels.

It should be clearly mentioned that the labels to be set on the **source cluster namespace** running the workloads.

Reply:
Fixed.